### PR TITLE
add new demo project from LVGL Square Line Studio

### PR DESCRIPTION
**Pull request by KunYi**
add new demo LVGL project 

demo picture in the below
image

### CODE REVIEW

**Comment by Rbb666** [comment]
感谢PR！

File: projects/hmi-board-ebike/README.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# 瑞萨 RA6M3-HMI-Board 开发板 BSP 说明
+
+## 简介
+
+本文档为 RA6M3-HMI-Board 开发板提供的 BSP (板级支持包) 说明。通过阅读快速上手章节开发者可以快速地上手该 BSP，将 RT-Thread 运行在开发板上。
```
Comment (Guozhanxin) [inline projects/hmi-board-ebike/README.md]: 这个readme需要更新一下，改成当前这个demo的介绍。

Comment (KunYi) [reply projects/hmi-board-ebike/README.md]: Got it, Let's me modify the README.md

**Comment by KunYi** [comment]
please check latest commit  '[1ce398fd](https://github.com/RT-Thread-Studio/sdk-bsp-ra6m3-hmi-board/pull/28/commits/1ce398fdd282aa16e5583f6e70ee6708cbb89f39)' for update README.md

**Review by Guozhanxin — APPROVED** [review]

**Pull request merged**